web/themes/custom/cloudy/src/js/back-to-top.js

```diff
@@ -1,27 +1,42 @@
-import $ from 'jquery';
-import Drupal from 'Drupal';
-
-Drupal.behaviors.cloudyBackToTop = {
-  attach: function(context, settings) {
-    var viewHeight = $(window).height();
-    var showHeight = viewHeight * 1.5;
-    var isAttached = false;
-
-    $(window).once('backToTopShowButtonHandler').on('scroll', function() {
-      var scrollPos = $(document).scrollTop();
-      if (scrollPos > showHeight && !isAttached) {
-        var buttonText = Drupal.t('Back to top');
-        $('#block-cloudy-content', context).append('<div id="back-to-top" class="btn cloudy-btn-purple"><a href="#header">' + buttonText + '</a></div>');
-        isAttached = true;
-      } else if (scrollPos <= showHeight && isAttached) {
-        $('#back-to-top').remove();
+/**
+ * @file
+ * Back to top button.
+ *
+ * Display back to top button at viewheight * 1.5.
+ */
+(function($, Drupal) {
+  Drupal.behaviors.cloudyBackToTop = {
+    attach: function(context, settings) {
+      var viewHeight = $(window).height();
+      var showHeight = viewHeight * 1.5;
+      var isAttached = false;
+  
+      $(window).once('backToTopShowButtonHandler').on('scroll', function() {
+        var scrollPos = $(document).scrollTop();
+        if (scrollPos > showHeight && !isAttached) {
+          var buttonText = Drupal.t('Back to top');
+  
+          // arrow
+          $('#block-cloudy-content', context).append('<div id="back-to-top" class="btn cloudy-btn-purple"><a href="#header">' + buttonText + '</a></div>');
+          
+          // single arrows
+          // $('#block-cloudy-content', context).append('<div id="back-to-top" class="btn cloudy-btn-purple"><a href="#header"><i class="fa-solid fa-chevron-up"></i></a></div>');
+  
+          // double arrows
+          // $('#block-cloudy-content', context).append('<div id="back-to-top" class="btn cloudy-btn-purple"><a href="#header"><i class="fa-solid fa-angles-up"></i></a></div>');
+          // $('#block-cloudy-content', context).append('<div id="back-to-top" class="btn cloudy-btn-purple"><a href="#header"> + buttonText + </a></div>');
```

Comment:
Now that we've decided to revert back to the text label, do we want to remove these commented out icon alternatives?

Reply:
Yes, I will take care of that